keys.go

```diff
@@ -263,6 +275,54 @@ func (c *Client) GetKeys(ctx context.Context, limit int, offset int) (*Keys, err
 	return &keys, nil
 }
 
+//ListKeysOptions struct to add the query parameters for the List Keys function
+type ListKeysOptions struct {
+	Extractable *bool
+	Limit       *uint64
+	Offset      *uint64
+	State       []int
```

Comment:
We should have or introduce an enum that can be used as a reference and also a convenient way of setting the State here to readable values, i.e.
```
ListKeysOptions{..., state: {States.ACTIVE}}
```